src/node.ts

```diff
@@ -57,16 +62,18 @@ export default class Node {
   public async init(): Promise<void> { // TODO: Check if this is working
     this.tor = await this.spawnTor()
     const onionAddress = await this.spawnService()
-    console.log('onion', onionAddress)
     const dataServer = await this.initDataServer()
     const connectonsManager = await this.initConnectionsManager(dataServer)
     const communities = new CommunitiesManager(connectonsManager)
     const peerId = await this.getPeer()
+    // eslint-disable-next-line
+    const certs = this.certificates
```

Comment:
it's bit weird you needed to disable linter for this line. What is the issue here?

Reply:
Btw. you don't need to assign class member to a local variable